rules-reviewed/quarkus/springboot/tests/data/springboot-jmx/SpringbootAnnotationConfiguration.java

```diff
@@ -0,0 +1,20 @@
+package org.springframework.jmx;
+
+import org.springframework.jmx.export.annotation.ManagedResource;
+import org.springframework.jmx.export.annotation.ManagedAttribute;
+
+@ManagedResource(objectName="bean:name=jmxTestBean", description="Test Bean")
+public class JMXAnnotationTestBean implements IJmxTestBean {
+
+    private String item;
+
+    @ManagedAttribute(description="Item", currencyTimeLimit=15)
+    public int getItem() {
+        return item;
+    }
+
+    public void setItem(int item) {
+        this.item = item;
+    }
+
+}
```

Comment:
Better to add newline at end of file : https://thoughtbot.com/blog/no-newline-at-end-of-file